src/components/Counter.js

```diff
@@ -6,11 +6,18 @@ class Counter extends Component {
     incrementIfOdd = () => {
         // Stretch Problem: Implement an increment function that
         // only increments if the counter value is odd
+        if(!(this.props.count %2 == 0)){
```

Comment:
What might be better for readability here is:
`if (this.props.count !== 0) { // ...`